e2e-tests/cypress.json

```diff
@@ -0,0 +1,18 @@
+{
+  "baseUrl": "https://atb-webshop-staging.web.app",
+  "integrationFolder": "cypress/integration/tests",
+  "watchForFileChanges": false,
+
+  "retries": {
+    "runMode": 1,
+    "openMode": 0
+  },
+
+  "env": {
+    "a11yTags": ["wcag2a", "wcag2aa"],
+    "a11ySkipFailures": false,
+    "email": "tor.moseng+test3@gmail.com",
+    "password": "qwer1234",
```

Comment:
Kunne sikkert også lagt inn dette som en annen miljøvariabel som bruk via GH Actions. Noe merarbeid og det er ikke krise at den ligger her ettersom det er staging/test

Reply:
Ja, tenkte på samme, men gikk for denne for å komme raskere til målet - og som du sier så er det testmiljø.